### ISSUE TITLE
[1.20.1] Compacting Drawers not using stone tag

### ISSUE THREAD

**Issue by vizthex123** [Buuz135/FunctionalStorage#284]
I noticed that the other blocks like them use the `forge:stone` tag, but the Compacting Drawers don't.

**Functional Storage Version:**
1.2.10

**Titanium Version:**
3.8.28

**Comment by Buuz135** [comment]
What?

**Comment by vizthex123** [comment]
The recipes don't use the `forge:stone` tag, which is a bit annoying since they'll only accept vanilla Stone.

**Comment by ffuentesm** [comment]
I think that is the intent. 
The recipe asks for Vanilla Stone, not any stone block.

**Comment by vizthex123** [comment]
yes, but it should use the tag to have better mod compatibility.

**Comment by ffuentesm** [comment]
Forge:Stone encompasses, Stone, cobblestone, deepslate variants, All Polish stones.
![image](https://github.com/user-attachments/assets/ec35bf88-7645-483b-be54-f22dd5095615)
So having it be forge:stone would mean that stone is kind of Pointless since you could use cobblestone.
